services/ui-src/src/components/statusing/StatusIcon.tsx

```diff
@@ -0,0 +1,54 @@
+import { Flex, Image, Text } from "@chakra-ui/react";
+import { ReportType } from "types";
+// assets
+import successIcon from "assets/icons/icon_check_circle.png";
+import errorIcon from "assets/icons/icon_error_circle_bright.png";
+import { assertExhaustive } from "utils/other/typing";
+
+export const StatusIcon = ({
+  reportType,
+  status,
+}: {
+  reportType: ReportType;
+  status?: boolean;
+}) => {
+  switch (reportType) {
+    case ReportType.SAR:
+    case ReportType.WP: {
```

Comment:
could we refactor this to remove the switch statement if we treat both report types the same? could just be an if/else to return.

I'm not sure I understand the use case for the default error case 🤔 are we worried about having status icons hooked up while building a report?

Reply:
I think that the switch is logic from MCR, where the statuses of the report types are treated differently. And the default I think was added during development of new reports. I could be wrong though. I'll remove the switch here